Tasks/DockerComposeV0/dockercomposebuild.ts

```diff
@@ -47,32 +51,40 @@ function addOtherTags(connection: DockerComposeConnection, imageName: string): a
     function addLatestTag() {
         var includeLatestTag = tl.getBoolInput("includeLatestTag");
         if (baseImageName !== imageName && includeLatestTag) {
-            return dockerTag(connection, imageName, baseImageName);
+            return dockerTag(connection, imageName, baseImageName, outputUpdate);
         }
     }
 
     var promise = addAdditionalTags();
     promise = !promise ? addSourceTags() : promise.then(addSourceTags);
     promise = !promise ? addLatestTag() : promise.then(addLatestTag);
+    
+
+    if (promise) {
+        //Only execute this is the promise was set. Else there's no output
+        
+    }
```

Comment:
What are you executing in this block?

Reply:
This was temporary code from when I was testing some things, good catch, I removed it :smile: